hw03_smithj7.py

```diff
@@ -0,0 +1,103 @@
+
+
+
+#################################################################################
+# Author: Juno Smith
+# Username: smithj
+#
+# Assignment: Hw03
+# Purpose: HW
+# Google Doc Link:https://docs.google.com/document/d/12MjBFSdEOX1sSblb_gTghHuoPa7JGxCLgrh2dcQSNbs/edit
+#
+#################################################################################
+# Acknowledgements:
+# Happy meal
+#
+#################################################################################
+import turtle
+turtle.colormode(255)
```

Comment:
There should be no code outside of functions. This probably belongs at the start of `main()`